doc/write_your_own/connections/filePickerSimple.ipynb

```diff
@@ -56,7 +56,7 @@
   },
```

Comment:
This demonstrates how to "select multiple files" yet the result in the last line only contains one file?